analyzers/src/SonarAnalyzer.Common/Facade/ILanguageFacade.cs

```diff
@@ -22,26 +22,25 @@
 using Microsoft.CodeAnalysis;
 using SonarAnalyzer.Helpers.Facade;
 
-namespace SonarAnalyzer.Helpers
+namespace SonarAnalyzer.Helpers;
+
+public interface ILanguageFacade
 {
-    public interface ILanguageFacade
-    {
-        AssignmentFinder AssignmentFinder { get; }
-        StringComparison NameComparison { get; }
-        StringComparer NameComparer { get; }
-        GeneratedCodeRecognizer GeneratedCodeRecognizer { get; }
-        IExpressionNumericConverter ExpressionNumericConverter { get; }
+    AssignmentFinder AssignmentFinder { get; }
+    StringComparison NameComparison { get; }
+    StringComparer NameComparer { get; }
+    GeneratedCodeRecognizer GeneratedCodeRecognizer { get; }
+    IExpressionNumericConverter ExpressionNumericConverter { get; }
 
-        DiagnosticDescriptor CreateDescriptor(string id, string messageFormat, bool? isEnabledByDefault = null, bool fadeOutCode = false);
-        object FindConstantValue(SemanticModel model, SyntaxNode node);
-        IMethodParameterLookup MethodParameterLookup(SyntaxNode invocation, IMethodSymbol methodSymbol);
-    }
+    DiagnosticDescriptor CreateDescriptor(string id, string messageFormat, bool? isEnabledByDefault = null, bool fadeOutCode = false);
+    object FindConstantValue(SemanticModel model, SyntaxNode node);
+    IMethodParameterLookup MethodParameterLookup(SyntaxNode node, IMethodSymbol methodSymbol);
```

Comment:
I would keep `invocation` here. This is language-agnostic, so we can not be more specific with the type, but the parameter name should reflect, that we expect some kind of invocation expression here
```suggestion
    IMethodParameterLookup MethodParameterLookup(SyntaxNode invocation, IMethodSymbol methodSymbol);
```